.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: go
 
+go:
+  - "1.13"
```

Comment:
I think we can safely jump to 1.15

Reply:
We have "You will also need a recent version of Go (we recommend at least Go 1.13 at the moment)." in Readme, so I thought that 1.13 is a minimal supported version. If so, I think it's wise to keep tests running the minimal version. :)

Reply:
We can also bump the one in the README :) I'm definitely not willing to support Go < 1.15 at the moment